lib/export/formatters/dataElementFormatterCimpl6.js

```diff
@@ -595,6 +601,7 @@ class DataElementFormatterCimpl6 {
     case 'TBD':
       return `TBD "${value.text}"`;
     default:
+    console.log('inside the default option')
```

Comment:
I'm guessing you did not mean to leave this active.

Reply:
Yup, was using it for debugging. Have removed it now.